src/opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -154,16 +155,17 @@ AquiferConfig load_aquifers(const Deck& deck, const TableManager& tables, NNC& i
 
         const auto& init_config = this->getInitConfig();
         if (init_config.restartRequested()) {
+            const auto& restart_keyword = deck.getKeyword<ParserKeywords::RESTART>();
             const auto& io_config = this->getIOConfig();
             const int report_step = init_config.getRestartStep();
             const auto& restart_file = io_config.getRestartFileName( init_config.getRestartRootName(), report_step, false);
             if (!std::filesystem::exists(restart_file))
-                throw std::logic_error(fmt::format("The restart file: {} does not exist", restart_file));
+                throw OpmInputError(fmt::format("ERROR: The restart file: {} does not exist", restart_file), restart_keyword.location());
```

Comment:
Please remove ERROR:, as the message is already reported as Error two lines above.